components/BattleScreen/BattleScreen.js

```diff
@@ -15,13 +14,18 @@ export default function BattleScreen() {
 	const [victory, setVictory] = useState(false);
 	const [isDisabled, setIsDisabled] = useState(false);
 	const [isEnemyDefeated, setIsEnemyDefeated] = useState(false);
+	const [playerAttacking, setPlayerAttacking] = useState(false);
+	const [enemyAttacking, setEnemyAttacking] = useState(false);
+
+	console.log(enemyHealth);
 
-	const handleAttack = () => {
+	const handleAttack = (damage) => {
 		// Ich greife an
```

Comment:
If you need this comment, you should consider renaming the function; what about `handlePlayerAttack` or similar?